Dockerfile.hip

```
@@ -3,7 +3,7 @@
 # Distributed under the Boost Software License, Version 1.0. (See accompanying
 # file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
 
-FROM stellargroup/build_env:10
+FROM pikaorg/pika-ci-base:1
```

```suggestion
ARG BASE_IMAGE # maybe =pikaorg/pika-ci-base:latest, but maybe no default
FROM $BASE_IMAGE
```
and then you could pass the base image with `build-arg`?